spacewalk/config/usr/share/man/man5/rhn.conf.5

```diff
@@ -215,13 +215,20 @@ then the job will be queued.
 2
 
 .TP
-.B "taskomatic.maxmemory" (integer)
+.B "taskomatic.java.maxmemory" (integer)
```

Comment:
Hey Tomas,

No, the taskomatic.java.[init/max]memory is correct, I think because rhn.conf does not know that wrapper.java.maxmemory would refer to Tasko's memory (just a guess). For actual proof:

[root@sat58mmello ~]# grep memory /etc/rhn/rhn.conf
taskomatic.java.initmemory=512
taskomatic.java.maxmemory=1192

[root@sat58mmello ~]# service taskomatic stop
Stopping RHN Taskomatic...
Stopped RHN Taskomatic.

[root@sat58mmello ~]# ps aux | grep memory | head -1
root      4285  0.0  0.0  19132   672 ?        Sl   18:06   0:00 /usr/bin/taskomaticd /usr/share/rhn/config-defaults/rhn_taskomatic_daemon.conf wrapper.java.initmemory=512 wrapper.java.maxmemory=1192 wrapper.pidfile=/var/run//taskomatic.pid wrapper.daemonize=TRUE

  ^ it worked